styleguide/source/_patterns/01-molecules/06-components/20-card-course/card-course.twig

```diff
@@ -0,0 +1,10 @@
+<a class="card card-course" href="#">
+  {% include 'atoms-landscape-3x2' with { 'class': 'card-course_image' } %}
+  <div class="card-course_inner">
+    {% include 'atoms-paragraph' with { 'class': 'card-course_duration', 'content' : '1hr 15min' } %}
+    {% include 'atoms-h2' with { 'class': 'card-course_title', 'content' : 'Association Between Elevated Brain Amyloid and Subsequent Cognitive Decline Among Cognitively Normal Persons' } %}
+    {% include 'atoms-paragraph' with { 'class': 'card-course_credits', 'content' : '3.00 CME Credits'  } %}
```

Comment:
In the wireframes it looks like the credit hours are bold like the title and start date. Should they be bold here or do we have a reason to prefer they be normal font weight?